### ISSUE TITLE
Code review - lessons learned, keeping track of who is 'up'

### ISSUE THREAD

**Issue by bpbond** [PecanProject/pecan#883]
Hi all, it seems like you use a round-robin system for code review. Have you found this valuable? Importantly, how do contributors know whose 'turn' it is? I'm asking on behalf of a similarly-sized science software project that is trying to implement a mechanism for ensuring all PRs get reviewed.

Thanks!

**Comment by mdietze** [comment]
@bpbond we've got a partial system -- specifically, my lab internally has a round-robin code review that we started at the beginning of the year as part of a lab-wide annual review we do every January 1.  That includes everyone in the lab for whom PEcAn is their primary repo (large majority) plus @crollinson's PalEON repos plus whatever Colin Averill's working on at the moment.  We just have a Google spreadsheet for that. As you've noticed the person submitting the PR just tags the next person on their list.

For folks on PEcAn that are outside my lab, the system is more informal -- folks look based on personal interest and either myself, Rob, Tony, or David sign off.
